lbz/dev/server.py

```diff
@@ -110,15 +110,8 @@ def handle_request(self) -> None:
                     body=request_obj,
                 )
             )
-            response = resource()
-            code = response.status_code
-            response_as_dict = response.to_dict()
```

Comment:
There is a piece of logic inside the `.to_dict()` method which makes sense to me but you skipped it and you always returns raw `body`:
```python
body = (
    json.dumps(self.body, separators=(",", ":"), default=str)
    if self.is_json
    else self.body
)
```
What do you think about that? 🤔